tests/test_requirements.py

```diff
@@ -1195,3 +1200,21 @@ def test_try_star_requirements():
         "otherwise",
         "finish",
     ]
+
```

Comment:
The other side of requirements (what expressions depend on) is bindings (what expressions provide).  Would be nice to have a couple of tests to make sure we're just exporting the alias (plus any crazy walrus operator bindings).